components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java

```diff
@@ -266,6 +268,7 @@ public void updateConsumerApplication(OAuthConsumerAppDTO consumerAppDTO) throws
         OAuthAppDO oauthappdo = new OAuthAppDO();
         oauthappdo.setUserName(tenantAwareUsername);
```

Comment:
This was not going to database level and just kept in appInfoCache. Corrected to be consistent by removing domain name.